### PR TITLE
Update free-programming-books-subjects.md

### PR DESCRIPTION
**Pull request by ANMS3I3**
## What does this PR do?
Add resource

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -646,6 +646,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [HTTP2 Explained](http://daniel.haxx.se/http2/) - Daniel Stenberg
 * [Introduction to HTTP](https://launchschool.com/books/http) - Launch School
 * [IPv6 for IPv4 Experts](https://sites.google.com/site/yartikhiy/home/ipv6book) - Yar Tikhiy (PDF)
+* [ Introduction To Networking : How the internet works](http://do1.dr-chuck.net/net-intro/EN_us/net-intro.pdf) - Charles Severance (PDF)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:649]:
```suggestion
* [Introduction To Networking : How the internet works](http://do1.dr-chuck.net/net-intro/EN_us/net-intro.pdf) - Charles Severance (PDF)
```

**Comment by eshellman** [comment]
I think this is a duplicate

**Comment by ANMS3I3** [comment]
> I think this is a duplicate

How

**Comment by eshellman** [comment]
> > I think this is a duplicate
> 
> How

It seems to be listed under Open Source Ecosystem. Probably not the right place.

**Comment by ANMS3I3** [comment]
> > > I think this is a duplicate
> > 
> > 
> > How
> 
> It seems to be listed under Open Source Ecosystem. Probably not the right place.

So what should I do ? Close the pr ?

**Comment by eshellman** [comment]
either close the pr or delete the other listing if you feel that the "networking" subject is a better fit. Add an explanation to the PR. - I've not read the book.

**Pull request closed without merging**